WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewController.swift

```diff
@@ -66,14 +70,13 @@ struct SetUpTapToPayInformationView: View {
     var setUpButtonAction: (() -> Void)? = nil
     var showURL: ((URL) -> Void)? = nil
     var learnMoreUrl: URL? = nil
+    var dismiss: (() -> Void)? = nil
```

Comment:
We can add use `@Environment(\.dismiss)` in `SetUpTapToPayInformationView` to remove this closure and the dismiss logic from `configureView()`